test/intl402/DurationFormat/prototype/format/style-options-en.js

```diff
@@ -9,10 +9,10 @@ features: [Intl.DurationFormat]
 ---*/
 
 const testData = {
-  "long" : "1 year, 2 months, 3 weeks, 3 days, 4 hours, 5 minutes, 6 seconds, 7 milliseconds, 8 microseconds, and 9 nanoseconds",
-  "short": "1 yr, 2 mths, 3 wks, 3 days, 4 hr, 5 min, 6 sec, 7 ms, 8 μs, and 9 ns",
-  "narrow":"1y, 2m, 3w, 3d, 4h, 5m, 6s, 7ms, 8μs, and 9ns",
-  "digital":"1y, 2m, 3w, 3d, and 4:05:06.007",
+  "long" : "1 year, 2 months, 3 weeks, 3 days, 4 hours, 5 minutes, 6 seconds, 7 milliseconds, 8 microseconds, 9 nanoseconds",
```

Comment:
Is this line correct? By my reading of https://tc39.es/proposal-intl-duration-format/#sec-properties-of-intl-durationformat-constructor the list formatter is constructed with `{ style: 'long' }`, which produces a list with the format "A, B, and C"

Reply:
`DurationFormat` also sets the option `{ type : 'unit' }` when it constructs the `ListFormat` object. The CLDR does not use "and" in the list pattern for units:
https://github.com/unicode-org/cldr/blob/main/common/main/en.xml#L9107-L9112

Reply:
ah, got it, thanks.